EventsExpress.Core/NotificationHandlers/ApproveParticipantHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using EventsExpress.Core.DTOs;
+using EventsExpress.Core.Extensions;
+using EventsExpress.Core.IServices;
+using EventsExpress.Core.Notifications;
+using MediatR;
+
+namespace EventsExpress.Core.NotificationHandlers
+{
+    public class ApproveParticipantHandler : INotificationHandler<ApproveParticipantMessage>
```

Comment:
Code for this file and `DenyParticipanthandler` is duplicated. Consider extracting it into some base class.